CONTRIBUTING.md

```diff
@@ -43,15 +43,15 @@ To get a basic development environment for Go and k6 up and running, first make
 
 Once that's done, you can get the k6 source into your Go workspace (`$GOPATH/src`) by running:
 ```bash
-go get github.com/loadimpact/k6
+go get github.com/k6io/k6
 ```
 This will also build a `k6` binary and put it in `$GOPATH/bin`.
 
 **Building from source**:
 
-Standing in the repo root (`$GOPATH/src/github.com/loadimpact/k6`) you can build a k6 binary from source by running:
+Standing in the repo root (`$GOPATH/src/github.com/k6io/k6`) you can build a k6 binary from source by running:
```

Comment:
Not for this PR, but should we remove the `GOPATH` details from here? And from the `Dockerfile` and `README` too?

Reply:
yeah, probably good idea, but not in this PR